### PR TITLE
WINUI-3339 - UG for Avatar view - Overview

### PR DESCRIPTION
**Pull request by MadhumithanJayapathi**
### Description
Created Overview page and added the empty md files for all the remaining pages.

### CODE REVIEW

File: winui-toc.html [hunk 1]
```diff
@@ -85,6 +85,16 @@
 			<li><a href="/winui/AutoComplete/Highlighting">Highlighting</a></li>
         </ul>
     </li>
+    <li>
+        AvatarView
+       <ul>
```
Comment (Manivannan-E) [inline winui-toc.html:90]: @MadhumithanJayapathi Use small letters for href link.

File: winui/AvatarView/Content-Type.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn what are the content types in AvatarType and how to set value for the content types in WinUI AvatarView.
+platform: WinUI
+control: SfAvatarView
+documentation: ug
+---
+
```
Comment (Manivannan-E) [inline winui/AvatarView/Content-Type.md:9]: @MadhumithanJayapathi Content types in WinUI Avatar View

File: winui/AvatarView/Customization.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Customization in WinUI AvatarView control | Syncfusion
+description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# Customization in SfAvatarView
```
Comment (Manivannan-E) [inline winui/AvatarView/Customization.md:10]: @MadhumithanJayapathi Customization in WinUI AvatarView

File: winui/AvatarView/Overview.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+platform: WinUI
+control: SfAvatarView
+documentation: ug
+---
+
+# WinUI Avatar View (SfAvatarView) Overview
+
+The AvatarView control provides a graphical representation of user image that allows you to customize the view by adding image, background color, icon, text, etc.
+
+![AvatarView control overview in WinUI](AvatarView_images/winui_avatarview_overview.png)
```
Comment (Manivannan-E) [inline winui/AvatarView/Overview.md:14]: @MadhumithanJayapathi WinUI AvatarView control overview. use small letters for folder name of images.

File: winui/AvatarView/Overview.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of WinUI AvatarView control | Syncfusion
+description: Learn all about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more here.
+platform: WinUI
+control: SfAvatarView
+documentation: ug
+---
+
+# WinUI Avatar View (SfAvatarView) Overview
```
Comment (Manivannan-E) [inline winui/AvatarView/Overview.md:10]: @MadhumithanJayapathi WinUI AvatarView Overview

File: winui/AvatarView/Overview.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+The AvatarView control provides a graphical representation of user image that allows you to customize the view by adding image, background color, icon, text, etc.
+
+![AvatarView control overview in WinUI](AvatarView_images/winui_avatarview_overview.png)
+
+## Key features
+
+* Supports for adding image, initials and group.
```
Comment (Manivannan-E) [inline winui/AvatarView/Overview.md:18]: @MadhumithanJayapathi 

* `Content type`: Supports for adding image, initials and 25 predefined avatars.
* `Group view`: Add maximum three custom images or initials in a single view.
* `Visual styles`: Supports different types of visual styles such as circle, square and custom. 
* `Customization`: Customizes the Height, Width, BorderBrush, BorderThickness, Background, CornerRadius and Font.

File: winui-toc.html (hunk 1 above)
Comment (MadhumithanJayapathi) [reply winui-toc.html:90]: @Manivannan-E  Used small letters for href links

File: winui/AvatarView/Content-Type.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn what are the content types in AvatarType and how to set value for the content types in WinUI AvatarView.
```
[the remark this answers is not in the document]
Comment (MadhumithanJayapathi) [reply winui/AvatarView/Content-Type.md]: @Manivannan-E  Changed the description as mentioned.

File: winui/AvatarView/Content-Type.md (hunk 2 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/Content-Type.md]: @Manivannan-E Changed the title as mentioned.

File: winui/AvatarView/Overview.md (hunk 4 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/Overview.md]: @Manivannan-E Used small letters for folder name for images

File: winui/AvatarView/Overview.md (hunk 5 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/Overview.md]: @Manivannan-E Changed the title as mentioned.

File: winui/AvatarView/Overview.md (hunk 6 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/Overview.md]: @Manivannan-E Changed the key features as given

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/AvatarView/content-types.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/AvatarView/content-types.md:3]: Title: Content Types in WinUI AvatarView control | Syncfusion

File: winui/AvatarView/content-types.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+description: Learn what are the content types available and how to set value for the content types in WinUI AvatarView control.
```
Comment (Chrispineagunja) [inline winui/AvatarView/content-types.md:4]: Description: Learn the content types available and how to set a value for the content types in WinUI AvatarView control.

File: winui/AvatarView/content-types.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
...
+platform: WinUI
```
Comment (Chrispineagunja) [inline winui/AvatarView/content-types.md:5]: Platform: WinUI

File: winui/AvatarView/content-types.md [hunk 11]
```diff
@@ -0,0 +1,10 @@
...
+control: AvatarView
```
Comment (Chrispineagunja) [inline winui/AvatarView/content-types.md:6]: Control: AvatarView

File: winui/AvatarView/content-types.md [hunk 12]
```diff
@@ -0,0 +1,10 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline winui/AvatarView/content-types.md:7]: Documentation: UG

File: winui/AvatarView/customization.md [hunk 13]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline winui/AvatarView/customization.md:2]: Layout: Post

File: winui/AvatarView/customization.md [hunk 14]
```diff
@@ -0,0 +1,10 @@
...
+title: Customization in WinUI AvatarView control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/AvatarView/customization.md:3]: Title: Customization in WinUI AvatarView control | Syncfusion

File: winui/AvatarView/customization.md [hunk 15]
```diff
@@ -0,0 +1,10 @@
...
+description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
```
Comment (Chrispineagunja) [inline winui/AvatarView/customization.md:4]: Description: Learn about the customization features such as Height, Width, BorderBrush, Background, FontSize, and CornerRadius in WinUI AvatarView control.

File: winui/AvatarView/customization.md [hunk 16]
```diff
@@ -0,0 +1,10 @@
...
+platform: WinUI
```
Comment (Chrispineagunja) [inline winui/AvatarView/customization.md:5]: Platform: WinUI

File: winui/AvatarView/customization.md [hunk 17]
```diff
@@ -0,0 +1,10 @@
...
+control: AvatarView
```
Comment (Chrispineagunja) [inline winui/AvatarView/customization.md:6]: Control: AvatarView

File: winui/AvatarView/customization.md [hunk 18]
```diff
@@ -0,0 +1,10 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline winui/AvatarView/customization.md:7]: Documentation: UG

File: winui/AvatarView/getting-started.md [hunk 19]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:2]: Layout: Post

File: winui/AvatarView/getting-started.md [hunk 20]
```diff
@@ -0,0 +1,10 @@
...
+title: Getting Started with WinUIAvatar View control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:3]: Title: Getting Started with WinUIAvatar View control | Syncfusion

File: winui/AvatarView/getting-started.md [hunk 21]
```diff
@@ -0,0 +1,10 @@
...
+description: Learn all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control, its elements, and more here.
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:4]: Description: Learn about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control, its elements, and more.

File: winui/AvatarView/getting-started.md [hunk 22]
```diff
@@ -0,0 +1,10 @@
...
+platform: WinUI
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:5]: Platform: WinUI

File: winui/AvatarView/getting-started.md [hunk 23]
```diff
@@ -0,0 +1,10 @@
...
+control: AvatarView
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:6]: Control: AvatarView

File: winui/AvatarView/getting-started.md [hunk 24]
```diff
@@ -0,0 +1,10 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:7]: Documentation: UG

File: winui/AvatarView/overview.md [hunk 25]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:2]: Layout: Post

File: winui/AvatarView/overview.md [hunk 26]
```diff
@@ -0,0 +1,21 @@
...
+title: Overview of WinUI AvatarView control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:3]: Title: Overview of WinUI AvatarView control | Syncfusion

File: winui/AvatarView/overview.md [hunk 27]
```diff
@@ -0,0 +1,21 @@
...
+description: Learn all about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more here.
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:4]: Description: Learn about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more.

File: winui/AvatarView/overview.md [hunk 28]
```diff
@@ -0,0 +1,21 @@
...
+platform: WinUI
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:5]: Platform: WinUI

File: winui/AvatarView/overview.md [hunk 29]
```diff
@@ -0,0 +1,21 @@
...
+control: AvatarView
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:6]: Control: AvatarView

File: winui/AvatarView/overview.md [hunk 30]
```diff
@@ -0,0 +1,21 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:7]: Documentation: UG

File: winui/AvatarView/overview.md [hunk 31]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+# WinUI AvatarView Overview
+
+The .NET AvatarView control provides a graphical representation of user image that allows you to customize the view by adding image, background color, icon, text, etc.
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:12]: The .NET AvatarView control provides a graphical representation of a user image that allows you to customize the view by adding an image, background color, icon, text, and more.

File: winui/AvatarView/overview.md [hunk 32]
```diff
@@ -0,0 +1,21 @@
...
+
+![WinUI AvatarView control overview](avatarview_images/winui_avatarview_overview.png)
+
+## Key features
+
+* `Content type`: Display initials, images, and 25 pre-defined avatars.
+* `Group view`: Add up to three images or initials in a single view.
+* `Visual styles`: Use different types of visual styles such as circle, square and custom.
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:20]: - Visual styles: Use different visual styles like circle, square, and custom.

File: winui/AvatarView/overview.md [hunk 33]
```diff
@@ -0,0 +1,21 @@
...
+The .NET AvatarView control provides a graphical representation of user image that allows you to customize the view by adding image, background color, icon, text, etc.
+
+![WinUI AvatarView control overview](avatarview_images/winui_avatarview_overview.png)
+
+## Key features
+
+* `Content type`: Display initials, images, and 25 pre-defined avatars.
+* `Group view`: Add up to three images or initials in a single view.
+* `Visual styles`: Use different types of visual styles such as circle, square and custom. 
+* `Customization`: Customize the height, width, border brush, border thickness, background, corner radius and font.
```
Comment (Chrispineagunja) [inline winui/AvatarView/overview.md:21]: * `Customization`: Customize the height, width, border brush, border thickness, background, corner radius, and font.

File: winui/AvatarView/visual-styles.md [hunk 34]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline winui/AvatarView/visual-styles.md:2]: Layout: Post

File: winui/AvatarView/visual-styles.md [hunk 35]
```diff
@@ -0,0 +1,10 @@
...
+title: Visual Styles in WinUI AvatarView control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/AvatarView/visual-styles.md:3]: Title: Visual Styles in WinUI AvatarView control | Syncfusion

File: winui/AvatarView/visual-styles.md [hunk 36]
```diff
@@ -0,0 +1,10 @@
...
+description: This section will explain about the different types of visual styles that exists in WinUI AvatarView.
```
Comment (Chrispineagunja) [inline winui/AvatarView/visual-styles.md:4]: Description: This section will explain the different types of visual styles in WinUI AvatarView.

File: winui/AvatarView/visual-styles.md [hunk 37]
```diff
@@ -0,0 +1,10 @@
...
+platform: WinUI
```
Comment (Chrispineagunja) [inline winui/AvatarView/visual-styles.md:5]: Platform: WinUI

File: winui/AvatarView/visual-styles.md [hunk 38]
```diff
@@ -0,0 +1,10 @@
...
+control: AvatarView
```
Comment (Chrispineagunja) [inline winui/AvatarView/visual-styles.md:6]: Control: AvatarView

File: winui/AvatarView/visual-styles.md [hunk 39]
```diff
@@ -0,0 +1,10 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline winui/AvatarView/visual-styles.md:7]: Documentation: UG

**Pull request merged**